pkg/aws/permissions.go

```diff
@@ -68,25 +68,22 @@ func checkPermissionsUsingQueryClient(queryClient, targetClient *awsClient, poli
 		}
 	}
 
+	resp, err := client.iamClient.SimulatePrincipalPolicy(input)
```

Comment:
we use `SimulatePrincipalPolicyPages()` here to ensure we capture all results. If the returned results are too long AWS will paginate the response, without it we could risk not validating all permissions.

Reply:
OK, I'll revert here